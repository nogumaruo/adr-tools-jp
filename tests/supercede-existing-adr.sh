adr new First Record
adr new -s 1 Second Record
head -11 doc/adr/001-first-record.md
head -13 doc/adr/002-second-record.md

