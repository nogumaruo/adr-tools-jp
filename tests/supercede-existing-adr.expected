adr new First Record
doc/adr/001-first-record.md
adr new -s 1 Second Record
doc/adr/002-second-record.md
head -11 doc/adr/001-first-record.md
# 1. First Record

日付: 1992-01-12
決定者: @xxx, @xxx

## ステータス

廃止 by [2. Second Record](002-second-record.md)

## コンテキストと問題点の説明

head -13 doc/adr/002-second-record.md
# 2. Second Record

日付: 1992-01-12
決定者: @xxx, @xxx

## ステータス

提案中

廃止 [1. First Record](001-first-record.md)

## コンテキストと問題点の説明


